third_party/clang/iOS.cmake

```diff
@@ -15,7 +15,7 @@ set(DIST_ARCH ${IOS_ARCH})
 
 add_definitions(-DIOS)
 
-set(IOS_MIN_TARGET "11.0")
+set(IOS_MIN_TARGET "13.0")
 
 if(PLATFORM_NAME STREQUAL "iphonesimulator")
```

Comment:
These two blocks about simulator and catalyst can be deleted, because on now we're on 13.0 to begin with

Reply:
Fixed in 66f4b53